src/app/work-item/work-item-board/work-item-board.component.scss

```diff
@@ -57,6 +57,9 @@
               justify-content: space-between;
             }
           }//boardcard ended
+          .board-card-selected {
+            border: 3px solid #bee1f4;
+          }
         }//boardlaneWrapper ended
         .lane-header {
           text-align: $textCenter;
```

Comment:
Why use 'text-align: $textCenter' instead of 'text-align: center'? $textCenter always equals center, so I'm confused as to why this is a variable.